extensions/common/xwalk_external_extension.h

```diff
@@ -39,8 +40,8 @@ class XWalkExternalExtension : public XWalkExtension {
 
   bool Initialize();
 
-  void set_runtime_variables(const base::ValueMap& runtime_variables) {
-    runtime_variables_ = runtime_variables;
+  void set_runtime_variables(base::DictionaryValue::Storage* runtime_variables) {
+      runtime_variables_.swap(*runtime_variables);
```

Comment:
Neither of them work. The thing is that there is base::ValueMap now. There is a using Storage = std::map<std::string, std::unique_ptr<Value>>; now. That is why I am using base::DictionaryValue::Storage now. 

What is more, it cannot be assigned (../../xwalk/extensions/common/xwalk_external_extension.h:45:23: error: no viable overloaded '='
   runtime_variables_ = runtime_variables;) or moved as long as it is not known how to copy unique_ptr unless operator= is overloaded.